flink-table-store-common/pom.xml

```diff
@@ -46,6 +46,12 @@ under the License.
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-shaded-jackson</artifactId>
```

Comment:
Can jackson be used here instead of `flink-shaded-jackson`?

Reply:
We can introduce jackson shaded by table store later.